docs/changelog.md

```diff
@@ -1,3 +1,26 @@
+## v2.5.0next [20xx-xx-xx]
+
+### Added
+- [FOGL-8942]: The ordering of the tabs in configuration user interface is no longer alphabetical, but controlled by the configuration category itself.[#471](https://github.com/fledge-iot/fledge-gui/pull/471)
+- [FOGL-8990]: The GUI rendering of lists in the configuration options has been improved.[#462](https://github.com/fledge-iot/fledge-gui/pull/462)
```

Comment:
GUI rendering of lists/kvlist?

Reply:
Fine in Added section, but what is improved?